src/components/NavigationBar.tsx

```diff
@@ -33,25 +34,19 @@ const NavigationBar: React.SFC<INavigationBarProps> = props => (
         <NavbarHeading className="hidden-xs">Source Academy</NavbarHeading>
       </NavLink>
 
-      <NavLink
-        activeClassName="pt-active"
-        className="NavigationBar__link pt-button pt-minimal"
-        target="_blank"
-        to={LINKS.LUMINUS}
-      >
-        <Icon icon={IconNames.BOOK} />
-        <div className="navbar-button-text hidden-xs">News &amp; Material</div>
-      </NavLink>
+      <a className="NavigationBar__link" href={LINKS.LUMINUS} target="_blank">
```

Comment:
Add `rel="noopener noreferrer"`

Refer to https://stackoverflow.com/a/45902400

Reply:
Ok done!